### PR TITLE
0.21.0 rc

### PR DESCRIPTION
**Pull request by AlexeyVavilin**
- ABI version 2 supported
- Message expiration implemented

### CODE REVIEW

File: src/tests/contracts/abi_v2/Giver.abi.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "ABI version": 1,
```
Comment (diserere) [inline src/tests/contracts/abi_v2/Giver.abi.json:2]: Shouldn't there be ABI v2 here?

File: src/tests/contracts/abi_v2/Wallet.abi.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "ABI version": 1,
```
Comment (diserere) [inline src/tests/contracts/abi_v2/Wallet.abi.json:2]: Shouldn't there be ABI v2 here?

File: src/tests/contracts/abi_v2/Giver.abi.json (hunk 1 above)
Comment (AlexeyVavilin) [reply src/tests/contracts/abi_v2/Giver.abi.json]: Done

File: src/tests/contracts/abi_v2/Wallet.abi.json (hunk 2 above)
Comment (AlexeyVavilin) [reply src/tests/contracts/abi_v2/Wallet.abi.json]: Done

**Comment by AlexeyVavilin** [comment]
Can not build actual contracts with current compiler version. Better do it in next release

**Pull request merged**